quiche/src/lib.rs

```diff
@@ -14190,6 +14236,58 @@ mod tests {
             .app_limited());
     }
 
+    #[test]
+    fn tx_cap_factor() {
```

Comment:
Is it worth testing the different cc algorithms?

Reply:
The send capacity is applied independently of CC implementation, so I don't think it would provide much added coverage TBH.